prestoadmin/util/version_util.py

```diff
@@ -19,102 +19,58 @@
 import re
 
 TD_VERSION = re.compile(r'^\d+t$')
-ANCIENT_TAGGED_VERSION = re.compile(r'^\d+t?-[A-Z]+$')
 
 
 def split_version(version_string):
-    return version_string.strip().split('.')
+    # We split on '.' and '-' because ancient tagged versions had the tag
+    # delimited by a '-'
+    return re.split('\.|-', version_string.strip())
 
 
 # New versions of the Teradata RPM are of the form 0.148.t.x.y, so the 't'
```

Comment:
I think the comment is talking about post 0.123t, but it's still not that helpful.  I'll get rid of it.